libraries/common/effekt.effekt

```diff
@@ -604,11 +604,31 @@ def each(start: Int, end: Int) { action: (Int) => Unit / Control } = {
 
 def repeat(n: Int) { action: () => Unit / Control } = each(0, n) { n => action() }
 
-
-// For boxing polymorphic values
-record BoxedUnit(unboxUnit: Unit)
-record BoxedInt(unboxInt: Int)
-record BoxedChar(unboxChar: Char)
-record BoxedBool(unboxBool: Bool)
-record BoxedDouble(unboxDouble: Double)
-record BoxedString(unboxString: String)
\ No newline at end of file
+namespace internal {
+  namespace boxing {
+    // For boxing polymorphic values
+    extern type BoxedUnit
+    extern pure def boxUnit(u: Unit): Unit =
+    extern pure def unboxUnit(u: Unit): Unit =
```

Comment:
Remember that we do not need to box Unit and Boolean (and hopefully soon String) in LLVM

Reply:
As for `String`: That means that we still need to box them?

Reply:
They are represented as `c_buffer` at the moment, which is not (yet) a `Pos`. We make it one though, which should be easy. Better box it here and we remove the boxing the moment this is fixed.